### ISSUE TITLE
error: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by WhyNotHugo** [mitsuhiko/rye#27]
```console
> rye toolchain register /usr/bin/python3
error: No such file or directory (os error 2)
```

**Comment by WhyNotHugo** [comment]
With https://github.com/mitsuhiko/rye/pull/26 I get:

```console
> cargo run --  toolchain register /usr/bin/python3
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
     Running `target/debug/rye toolchain register /usr/bin/python3`
Error: could not symlink interpreter

Caused by:
    No such file or directory (os error 2)
```

**Comment by WhyNotHugo** [comment]
Ah, I think the directory where the symlink is to be created does not exist.

**Comment by mitsuhiko** [comment]
@WhyNotHugo this should be resolved with 01fd9601e5f1ee5a5e708f6a43ceed1fe0d42de2

**Comment by WhyNotHugo** [comment]
Yup, we reached the same fix I see. Thanks!
